tests/loader/test_astronet_loader.py

```diff
@@ -0,0 +1,164 @@
+"""Unit tests for the COLMAP Loader class.
+
+Authors: John Lambert
+"""
+
+import unittest
+from pathlib import Path
+
+import numpy as np
+import trimesh
+from gtsam import Pose3, PinholeCameraCal3Bundler
+
+from gtsfm.common.image import Image
+from gtsfm.loader.astronet_loader import AstroNetLoader
+import gtsfm.utils.io as io_utils
+
+TEST_DATA_ROOT = Path(__file__).resolve().parent.parent / "data"
+TEST_POINT3D_IDS = [7, 2560, 3063, 3252, 3424, 3534, 3653, 3786, 3920, 4062, 4420, 4698, 3311, 3963, 325]
+TEST_SFMTRACKS_INDICES = [0, 100, 200, 300, 400, 500, 600, 700, 800, 900, 1000, 1100, 1200, 1300, 1400]
```

Comment:
could you add an inline comment explaining where these come from and how they were selected?

Reply:
Explanation added.

Reply:
If these have a 1:1 correspondences, it might be better to use a list of tuples instead of 2 lists. Ive found that to cause fewer bugs.

Reply:
They're used individually as well, so I thought it would be clearer to split them up.